skipper.go

```diff
@@ -1288,6 +1291,24 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
 	proxy := proxy.WithParams(proxyParams)
 	defer proxy.Close()
 
+	for _, startupCheckURL := range o.StartupChecks {
+		for {
+			/* #nosec */
+			resp, err := http.Get(startupCheckURL)
+			if err != nil {
+				log.Infof("%s unhealthy", startupCheckURL)
+				time.Sleep(1 * time.Second)
+				continue
+			}
+			if resp.StatusCode == 200 {
+				log.Infof("%s got healthy", startupCheckURL)
```

Comment:
"got healthy": maybe it was healthy in the beginning. Let's only log the success if it was unhealthy before.

Reply:
Does it matter?
I would like to see if there was a check that it actually checked it.

Reply:
then please change it from "got health" to "is healthy"

Reply:
I drop "got"